### PR TITLE
Add thunk tests

### PR DESCRIPTION
**Pull request by baldm0mma**

### CODE REVIEW

File: src/thunks/getMoviesThunk.test.js [hunk 1]
```diff
@@ -1,52 +1,49 @@
...
     mockDispatch = jest.fn();
-    window.fetch = jest.fn().mockImplementation(() => Promise.resolve({
+    window.fetch = jest.fn().mockImplementation(() =>
+    Promise.resolve({
       ok: true,
-      json: () => Promise.resolve({
-        movie: mockData.mockMovie
-      })
-    }))
+      json: () =>
```
Comment (pgoulding) [inline src/thunks/getMoviesThunk.test.js:16]: much thunking going on here

File: src/thunks/getMoviesThunk.test.js [hunk 2]
```diff
@@ -1,52 +1,49 @@
...
   });
 
   it('should dispatch throwError with a message if the response is not ok', async () => {
-    window.fetch = jest.fn().mockImplementation(() => Promise.resolve({
-      ok: false,
-      statusText: 'error'
-    }))
-    await thunk(mockDispatch)
-    expect(mockDispatch).toHaveBeenCalledWith(throwError('error'))
+    window.fetch = jest.fn().mockImplementation(() =>
```
Comment (pgoulding) [inline src/thunks/getMoviesThunk.test.js:35]: good test, much like.

File: src/thunks/getMoviesThunk.test.js [hunk 3]
```diff
@@ -1,52 +1,49 @@
...
+    await thunk(mockDispatch);
+    expect(mockDispatch).toHaveBeenCalledWith(throwError('error'));
   });
 
-  //need to make test for parsed response, cleaned data, and that movies was update - updateMovies
-
   it('should dispatch and not load if the response is ok', async () => {
-    await thunk(mockDispatch)
-    expect(mockDispatch).toHaveBeenCalledWith(isLoading(false))
+    await thunk(mockDispatch);
```
Comment (pgoulding) [inline src/thunks/getMoviesThunk.test.js:46]: nice job awaiting your thunks

File: src/thunks/getMoviesThunk.js [hunk 4]
```diff
@@ -1,32 +1,24 @@
...
       const data = await response.json();
-      console.log('success');
       const movieData = data.results;
       const movies = await cleanMovies(genre, movieData);
       dispatch(updateMovies(movies, genre));
       dispatch(isLoading(false));
       return Promise.all(movies);
     } catch (error) {
-      dispatch(throwError(error));
+      dispatch(throwError('error'));
```
Comment (pgoulding) [inline src/thunks/getMoviesThunk.js:19]: good error testing here

File: src/thunks/getMoviesThunk.js [hunk 5]
```diff
@@ -1,32 +1,24 @@
-import {
-  isLoading,
-  updateMovies,
-  throwError
-} from '../actions';
-import {
-  cleanMovies
-} from '../utils/cleanMovies';
+import { isLoading, updateMovies, throwError } from '../actions';
+import { cleanMovies } from '../utils/cleanMovies';
```
Comment (pgoulding) [inline src/thunks/getMoviesThunk.js:2]: great job on these imports, solid work

File: src/thunks/getMoviesThunk.test.js (hunk 1 above)
Comment (baldm0mma) [reply src/thunks/getMoviesThunk.test.js:16]: Lots. 4 realz.

File: src/thunks/getMoviesThunk.test.js (hunk 2 above)
Comment (baldm0mma) [reply src/thunks/getMoviesThunk.test.js:35]: Thunks.

File: src/thunks/getMoviesThunk.test.js (hunk 3 above)
Comment (baldm0mma) [reply src/thunks/getMoviesThunk.test.js:46]: Appreciate it.

File: src/thunks/getMoviesThunk.js (hunk 4 above)
Comment (baldm0mma) [reply src/thunks/getMoviesThunk.js:19]: Agreed. I'm a machine.

File: src/thunks/getMoviesThunk.js (hunk 5 above)
Comment (baldm0mma) [reply src/thunks/getMoviesThunk.js:2]: I really appreciate your feedback.

**Review by pgoulding — APPROVED** [review]

**Pull request merged**